auto-value-moshi/build.gradle

```diff
@@ -39,8 +40,9 @@ dependencies {
   compileOnly libraries.incap
   compileOnly libraries.autoService
 
+  api libraries.autoValueAnnotations
```

Comment:
Why was this one necessary? Since this isn't supposed to be a public API to anyone other than the extension itself

Reply:
I was thinking because client projects use the `@AutoValue` annotation to enable this, making it part of the public API, but I supposed you'd get that by adding it to the client project's dependencies.  I can revert that.